### PR TITLE
Create report of absence and refactor reports controller, add field t…

### LINKED ISSUE #17: Reports - Absence report by month
Reports - Absence report by month

### PR DESCRIPTION
**Pull request by alexs**
Thanks to:

alanRazoMen <alanRazoMen@users.noreply.github.com>
AlejandroCuevasZepeda <AlejandroCuevasZepeda@users.noreply.github.com>
MrCano369x <canorueda2000@gmail.com>

# Description
What did you implemented/Why did you implemented this:
 - Create absence report
 - Refactor reports controller and improve views
 - Added field is_active to employee and improve index view
 - Add seeds to populate data for develop test.
 - General improves
 - Other minor changes

 Examples:
 - What?: Added new report and general improvements
 - Why?: Cos is requesting by readme, this pull close issue #17 

 # Testing
 We have pending to approve pr with our test, to view refer to #18 

 Example:
  - I've added some methods to get reports in employee model. I used Faker for a few random data.

  ## Screenshots
Screen Shot 2022-03-30 at 6 19 52

### CODE REVIEW

**Review by Frcomparan — COMMENTED** [review]
LGTM, I'll just leave some recommendations

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,25 @@
...
+        private_number: Faker::Number.number(digits: 6),
+        company_branch_id: CompanyBranch.last.id,
+        is_active: true
+    )
+}
+
+50.times{ |d|
+    employee_id = rand(Employee.first.id..Employee.last.id)
+    Attendance.create(employee_id: employee_id, check_in: Time.now.midnight-d.day+rand(5..8).hours, check_out: Time.now.midnight-d.day+rand(13..16).hours)
+}
```
Comment (Frcomparan) [inline db/seeds.rb:29]: Add an empty space at the end of the file as a good practice

File: app/helpers/reports_helper.rb [hunk 2]
```diff
@@ -1,2 +1,5 @@
 module ReportsHelper
-end
+    def today_to_check(data)
+        data ? "✅": "❌"
+    end
+end
```
Comment (Frcomparan) [inline app/helpers/reports_helper.rb:5]: Add an empty space at the end of the file as a good practice

Comment (alexs) [reply app/helpers/reports_helper.rb:5]: Thanks

**Review by EliasUR — APPROVED** [review]
lgtm

**Review by Frcomparan — APPROVED** [review]
LGTM

**Review by DiegoSteve — APPROVED** [review]
LGTM!!

**Pull request merged**